Documentation/sources/content-card-ui/tutorial/customizing-content-card-templates.md

```diff
@@ -0,0 +1,50 @@
+# Customizing Content Card Templates
```

Comment:
Can we call this `Customizing Content Card Templates style` or something so its clear that only the look of the UI elements can be changed but not their position itself